browser/views.py

```diff
@@ -681,27 +705,53 @@ def edit_investigator_record(request,person_id,research_id):
     person = get_object_or_404(Person, pk=person_id)
     research = get_object_or_404(Investigator, pk=research_id)
     template = "browser/investigator.html"
+    institution_id = None
+    if research.institution:
+        institution_id = research.institution.id
 
     # Retreive investigator data already stored in database and display to the user
     form = InvestigatorForm(initial={'subject': research.subject, 'role': research.role,
-                                             'year': research.year})
+                                             'year': research.year, 'institution_search': research.institution,
+                                     'institution_id': institution_id})
+
+    context = {
+        'form': form,
+        'person': person,
+        'investigator_data': research,
+        'type': 'edit_investigator'
```

Comment:
is that used anywhere?

Reply:
No it was not being used. Removed the <b>type</b> variable.